### PR TITLE
[SW-835] Refactor: Move active account getter to accounts composable

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/NameItem.vue [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
             v-show="canBeDefault"
-            :class="{ set: account.name === name }"
-            :disabled="account.name === name"
+            :class="{ set: activeAccount.name === name }"
+            :disabled="activeAccount.name === name"
             @click="setDefault"
           >
             {{
-              account.name === name
+              activeAccount.name === name
```
Comment (pstachel) [inline src/popup/components/NameItem.vue:34]: That comparison is made three times here - what are your thoughts of a computed property that will check that?

**Review by pstachel — APPROVED** [review]

File: src/popup/components/NameItem.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/NameItem.vue]: OK, I created `isDefault` computed property for this purpose.

**Comment by subhod-i** [comment]
@peronczyk Pipeline failing. If not ready to review then please mark this as a draft PR.

File: src/composables/accounts.ts [hunk 2]
```diff
@@ -20,6 +23,23 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
...
+  const activeAccountSimplexLink = computed(() => buildSimplexLink(activeAccount.value.address));
+
+  function getAccountByAddress(address: string): IAccount | undefined {
+    return accounts.value.find((acc) => acc.address === address);
+  }
+
+  function setActiveAccountByIdx(idx?: number) {
+    // TODO replace with updating local state after removing the Vuex
+    store.commit('accounts/setActiveIdx', +(accounts.value[idx || 0].idx || 0));
+  }
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/composables/accounts.ts]: Makes sense. Updated.

**Pull request merged**